src/main/kotlin/racing/Racing.kt

```diff
@@ -0,0 +1,25 @@
+package racing
+
+object Racing {
+    fun start(carNumber: Int, tryRound: Int): ArrayList<Array<Int>> {
+        val racingCars = Array<RacingCar>(carNumber) { RacingCar(tryRound) }
```

Comment:
Array보다는 Collection을 활용해보면 어떨까요? 어떤 이점이 있을까요?

```suggestion
        val racingCars = List(carNumber) { RacingCar(tryRound) }
```